tests/pipeline/test_factor.py

```diff
@@ -222,3 +222,29 @@ def test_rsi(self, seed_value, expected):
         rsi.compute(today, assets, out, test_data)
 
         check_allclose(expected, out)
+
+    @parameterized.expand([
+        # Test cases computed by doing:
+        # from numpy.random import seed, randn
+        # seed(seed_value)
+        # data = abs(randn(window_length, 3))
+        # expected = (data[-1] - data[0]) / data[0]
+        (100, array([-0.99581969, 0.78866002, 0.12724000]), 15),
```

Comment:
I baked in the expected results for RSI because talib is an optional zipline dependency, so I couldn't just call it directly in the test.  For this, we should just compute the expected value directly in the test.